applylist/factory.go

```diff
@@ -36,6 +36,23 @@ func (f *Factory) Create() ([]string, []string, []string, error) {
 	return applyList, blacklist, whitelist, nil
 }
 
+// purgeCommentsFromList iterates over the list contents and deletes comment
+// lines. A comment is a line whose first non-space character is #
+func (f *Factory) purgeCommentsFromList(rawList []string) ([]string, error) {
```

Comment:
this can never return anything but []string, nil right?  Why not remove the error and save an if statement?